image.yaml

```diff
@@ -1,25 +1,62 @@
-- name: operator-builder
-  version: 1.0.0
-  from: golang:1.14
-  description: Builder Image for the Operator
+- schema_version: 1
+  name: "operator-builder"
+  version: "1.0.0"
```

Comment:
should we use the product version or community version here ?

Reply:
probably better, let's use product version, on master er can keep then always 1 version ahead.